modules/programs/mpv.nix

```diff
@@ -54,6 +54,21 @@ in {
     programs.mpv = {
       enable = mkEnableOption "mpv";
 
+      scripts = mkOption {
+        default = [];
+        type = types.listOf types.package;
+        defaultText = "[]";
+        example = literalExample ''
+          [
+            pkgs.mpvScripts.mpris
+            (builtins.fetchurl "https://raw.githubusercontent.com/mpv-player/mpv/master/TOOLS/lua/autoload.lua")
```

Comment:
I think it might be best to remove this `fetchurl` example to avoid promoting impure configurations.

Alternative switching to the pure `fetchurl` function.